docs/application/dotnet/guides/nui/nui-components/Switch.md

```diff
@@ -0,0 +1,67 @@
+# Switch
+Switch is a common component that can be used as a selector.
+
+![SwitchProperty](./media/SwitchProperty.PNG)
+
+## Create with Property
+To create a switch using property, follow these steps:
+1. Create switch using the default constructor:
+
+```
+Switch[] utilitySwitch = new Switch[4];
+```
+
+2. Set the switch property:
+
+```

```

Comment:
Kindly check the global comment for code block language identifier and improper alignment with bullet point.